src/Microsoft.Azure.Batch.SoftwareEntitlement.Server/Microsoft.Azure.Batch.SoftwareEntitlement.Server.csproj

```diff
@@ -21,11 +21,6 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Content Include="wwwroot\index.html">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-    </Content>
-  </ItemGroup>
```

Comment:
Removing `index.html` file from the build would make it impossible to for a browser to access the `sestest server` as a connectivity test.